environments/test.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "test",
```

Comment:
This file will be removed in the final commit. We still need to update the Rakefile to generate an environment for openstack.